src/point_one/fusion_engine/messages/control.h

```diff
@@ -178,46 +178,101 @@ struct alignas(4) ResetRequest : public MessagePayload {
   /**
    * Perform a device hot start.
    *
-   * A hot start is typically used to restart the navigation engine in a
-   * deterministic state, particularly for logging purposes.
+   * This will reset the navigation engine into a known state, using previously
+   * stored position and time information. The device will begin navigating
+   * immediately if possible.
    *
    * To be reset:
    * - The navigation engine (@ref RESTART_NAVIGATION_ENGINE)
-   * - All runtime data (GNSS corrections (@ref RESET_GNSS_CORRECTIONS), etc.)
    *
-   * Not reset:
+   * Not reset/performed:
+   * - All runtime data (GNSS corrections (@ref RESET_GNSS_CORRECTIONS), etc.)
    * - Position, velocity, orientation (@ref RESET_POSITION_DATA)
+   * - GNSS ephemeris data (@ref RESET_EPHEMERIS)
    * - Fast IMU corrections (@ref RESET_FAST_IMU_CORRECTIONS)
    * - Training parameters (slowly estimated IMU corrections, temperature
    *   compensation, etc.; @ref RESET_NAVIGATION_ENGINE_DATA)
    * - Calibration data (@ref RESET_CALIBRATION_DATA)
    * - User configuration settings (@ref RESET_CONFIG)
-   * - GNSS measurement engine (@ref RESTART_GNSS_MEASUREMENT_ENGINE)
+   * - Reboot GNSS measurement engine (@ref RESTART_GNSS_MEASUREMENT_ENGINE)
    * - Reboot navigation processor (@ref REBOOT_NAVIGATION_PROCESSOR)
    */
-  static constexpr uint32_t HOT_START = 0x000000FF;
+  static constexpr uint32_t HOT_START = 0x00000001;
```

Comment:
Ah, `control.h` looks ok or what I expected. So, the descriptions for `control.py` should match those for `control.h`.